pubsub.go

```diff
@@ -23,8 +23,8 @@ import (
 	timecache "github.com/whyrusleeping/timecache"
 )
 
-// DefaultMaximumMessageSize is 1mb.
-const DefaultMaxMessageSize = 1 << 20
+// DefaultMaxMessageSize is 2MB
+const DefaultMaxMessageSize = 1 << 21
```

Comment:
Is there a reason why `DefaultMaxMessageSize` is not a typed variable?

Reply:
Also, there is a second comment that should be changed: `// messages. The default value is 1MiB (DefaultMaxMessageSize).`

Reply:
If not specified, the default is int.
https://go.dev/play/p/IGETqnIyywz

Reply:
changed the comment